Sources/StytchCore/StytchClient/DFP/StytchClient+DFP.swift

```diff
@@ -13,7 +13,8 @@ public extension StytchClient {
                 throw StytchSDKError.consumerSDKNotConfigured
             }
             #if os(iOS)
-            return await dfpClient.getTelemetryId(publicToken: publicToken, dfppaDomain: dfppaDomain)
+            let telemetryId = await dfpClient.getTelemetryId(publicToken: publicToken, dfppaDomain: dfppaDomain)
+            return telemetryId
             #else
```

Comment:
The reason I did this for all the telemetry ids was so that if I needed to debug them it was far easier to insert a print statement.